doc/importexport.rst

```diff
@@ -32,6 +32,7 @@ Export Formats
 * VRML
 * VTP
 * 3MF
+* GLTF
```

Comment:
Where the documentation refers to the file format I'd suggest to change GLTF -> glTF.